tests/pass/target-spec-implies-target-feature.rs

```diff
@@ -0,0 +1,8 @@
+//! Ensure that the target features given in the target spec are actually enabled.
+//@only-target: armv7
+
+fn main() {
+    assert!(cfg!(target_feature = "v7"));
+    assert!(cfg!(target_feature = "vfp3"));
+    assert!(cfg!(target_feature = "thumb2"));
```

Comment:
`rustc --target armv7-unknown-linux-gnueabihf --print cfg` shows v7, vfp2 and thumb2, but not vfp3.

```
[...]
target_feature="aclass"
target_feature="dsp"
target_feature="fpregs"
target_feature="thumb2"
target_feature="v5te"
target_feature="v6"
target_feature="v6k"
target_feature="v6t2"
target_feature="v7"
target_feature="vfp2"
[...]
```